django_freeradius/tests/base/test_api.py

```diff
@@ -288,20 +288,26 @@ def test_accounting_list_200(self):
                           input_octets=4440909,
                           output_octets=1119074409))
         self.radius_accounting_model.objects.create(**data3)
-        response = self.client.get(self._acct_url)
-        self.assertEqual(len(response.json()), 3)
+        response = self.client.get(self._acct_url+"?page_size=1&page=1")
```

Comment:
use `'{0}?page_size=1&page=1'.format(self._acct_url)` which is more pythonic and in line with the rest of the style used in this package.